monitoring/prometheus/ymls/stage.yml

```diff
@@ -28,9 +28,9 @@
     scheme: 'http'
     metrics_path: '/metrics'
     static_configs:
-      - targets: ['discoveryprovider2.staging.audius.co:9091']
+      - targets: ['35.202.199.159:9091/']
         labels:
-          host: 'discoveryprovider2.audius.co'
+          host: 'discoveryprovider2.staging.audius.co'
           environment: 'stage'
```

Comment:
fine for now, in interest of shipping this - don't think we can expose port directly for prod like this though
we should assign a dns to this some point, and then do same for prod

Reply:
Sounds good - it also shouldn't be a problem if we end up using disovery node as a push gateway down the road.

Reply:
agreed but that won't happen anytime soon, for now we should get things working without it